### PR TITLE
[Docs]How to add your custom fixture suites?

### PR DESCRIPTION
**Pull request by Tomanhez**
| Q               | A
| --------------- | -----
| Branch?         | 1.4
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

### CODE REVIEW

File: docs/customization/suites.rst [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+                    order:
+                        options:
+                            amount: 20
+
+                    address:
+                        options:
+                            random: 10 # the number of addresses that are random created
+                            prototype:
+                                country_code: PL
+
```
Comment (lchrusciel) [inline docs/customization/suites.rst]: I'm missing where we are telling the user to call `bin/console sylius:fixtures:load poland` explicitly

File: docs/customization/suites.rst [hunk 2]
```diff
@@ -0,0 +1,259 @@
+Customizing Suites
+==================
+
+What are suites:
+~~~~~~~~~~~~~~~~
+Suites are a pocket fixtures that are defined in one place.
```
Comment (lchrusciel) [inline docs/customization/suites.rst]:
```suggestion
Suites are pocket of fixtures that can be run together.
```

File: docs/customization/suites.rst [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+
+Why would you customize a suites?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you need to create a configuration file where you define all necessarily configuration to your store.
+
+How to use suites?
+~~~~~~~~~~~~~~~~~~
+List suites are available after ``$ bin/console sylius:fixtures:list`` command.
```
Comment (lchrusciel) [inline docs/customization/suites.rst]:
```suggestion
Complete list of suites is can be shown with the ``$ bin/console sylius:fixtures:list`` command.
```

File: docs/customization/suites.rst [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+In Sylius default suites are loaded in ``$ bin/console sylius:fixtures:load``.
```
Comment (lchrusciel) [inline docs/customization/suites.rst]:
```suggestion
In the vanilla Sylius implementation, the ``default` suite is loaded if ``$ bin/console sylius:fixtures:load`` command is executed without any additional argument.
```

File: docs/customization/suites.rst [hunk 5]
```diff
@@ -0,0 +1,259 @@
...
+If you creating new suite you must use previous command with name of your suite: ``$ bin/console sylius:fixtures:load your_custom_suite``.
```
Comment (lchrusciel) [inline docs/customization/suites.rst]:
```suggestion
If you are creating a new suite you must use the previous command with the name of your suite: ``$ bin/console sylius:fixtures:load your_custom_suite``.
```

File: docs/customization/suites.rst [hunk 6]
```diff
@@ -0,0 +1,259 @@
...
+
+How to create suites?
+~~~~~~~~~~~~~~~~~~~~~
+
+.. tip::
+
+    You can browse the full implementation of this example on `this GitHub Pull Request
+    <https://github.com/Sylius/Customizations/pull/24>`__.
+
+If you want create your fixtures with different locale than en_US you must change parameter in ``src/config/services.yaml``.
```
Comment (lchrusciel) [inline docs/customization/suites.rst]:
```suggestion
.. warning::
	If you want create your fixtures with different locale than ``en_US`` you must change the ``locale`` parameter in ``src/config/services.yaml``.
	
	.. code-block:: yaml
	    parameters:
	        locale: pl_PL
```

Can you check, how does it look like?

File: docs/customization/suites.rst [hunk 7]
```diff
@@ -0,0 +1,259 @@
...
+
+.. code-block:: yaml
+
+    parameters:
+        locale: pl_PL
+
+Next create ``src/config/packages/fixtures.yaml`` and add below code it there.
```
Comment (lchrusciel) [inline docs/customization/suites.rst]:
```suggestion
**1.** Create a ``src/config/packages/fixtures.yaml`` file and add below code it there:
```

File: docs/customization/suites.rst [hunk 8]
```diff
@@ -0,0 +1,259 @@
...
+                        options:
+                            amount: 20
+
+                    address:
+                        options:
+                            random: 10 # the number of addresses that are random created
+                            prototype:
+                                country_code: PL
+
+Now you must call your custom suite with ``bin/console sylius:fixtures:load poland`` command.
```
Comment (lchrusciel) [inline docs/customization/suites.rst]:
```suggestion
**2.** Load your custom suite with ``$ bin/console sylius:fixtures:load poland`` command.
```

File: docs/customization/suites.rst (hunk 6 above)
Comment (Tomanhez) [reply docs/customization/suites.rst]: Screenshot 2019-08-26 at 10 17 01

File: docs/customization/suites.rst [hunk 9]
```diff
@@ -0,0 +1,261 @@
+Customizing Suites
+==================
+
+What are suites:
```
Comment (lchrusciel) [inline docs/customization/suites.rst]:
```suggestion
What are suites?
```

File: docs/customization/suites.rst [hunk 10]
```diff
@@ -0,0 +1,261 @@
...
+~~~~~~~~~~~~~~~~
+Suites are pocket of fixtures that can be run together.
```
Comment (lchrusciel) [inline docs/customization/suites.rst]:
```suggestion
Suites are pockets of fixtures that can be run together.
```

File: docs/customization/index.rst [hunk 11]
```diff
@@ -19,6 +19,7 @@ The Customization Guide is helpful while wanting to adapt Sylius to your persona
     state_machine
     grid
     tips_and_tricks
+    suites
```
Comment (lchrusciel) [inline docs/customization/index.rst]: I'm wondering if `suites` page is meaningful enough. Maybe we should add `fixtures` here, and `suites` would be one of the sections?

File: docs/customization/suites.rst [hunk 12]
```diff
@@ -0,0 +1,261 @@
+Customizing Suites
```
Comment (lchrusciel) [inline docs/customization/suites.rst]:
```suggestion
Customizing Fixture Suites
```

File: docs/customization/index.rst (hunk 11 above)
Comment (lchrusciel) [reply docs/customization/index.rst]: Let's only adjust `suites` naming and I think we should be fine
```suggestion
    fixture_suites
```

**Comment by lchrusciel** [comment]
Thanks, Tomasz! :tada:

**Pull request merged**